cts/cli/regression.dates.exp

```diff
@@ -83,7 +83,7 @@ Period: 2005-02-09 00:00:00Z to 2005-02-12 00:00:00Z
 =#=#=#= End test: '2005-040/2005-043' period - OK (0) =#=#=#=
 * Passed: iso8601               - '2005-040/2005-043' period
 =#=#=#= Begin test: '2005-040/2005-043' period (XML) =#=#=#=
-<pacemaker-result api-version="X" request="iso8601 --output-as=xml -p 2005-040/2005-043">
+<pacemaker-result api-version="X" request="iso8601 -p 2005-040/2005-043 --output-as=xml">
```

Comment:
This whole commit will change if we go with putting `--output-as=xml` after the first token